.chloggen/raid_metrics_in_hostreceiver.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: system
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Proposed metric semantics for a scraper in hostreceiver scraping raid metrics related to multiple device RAID disks
```

Comment:
Maybe make this note more generic, something like: Introduce storage attributes and metrics